extensions/wikia/AdEngine/src/ad-engine.bridge.js

```diff
@@ -57,6 +57,12 @@ function unifySlotInterface(slot) {
 	slot.config = slotContext;
 	slot.getVideoAdUnit = () => buildVastAdUnit(slot.name);
 	slot.getTargeting = () => slotContext.targeting;
+	slot.getElement = () => slot.container.parentElement;
```

Comment:
Are you sure with modifying an original slot object? Maybe something like `slot = Object.assign(new EventEmitter(), slot, { <NEW_METHODS_AND_PROPS> })` earlier will be better?